src/software/ai/hl/stp/tactic/test_tactics/move_test_tactic.cpp

```diff
@@ -38,3 +38,9 @@ void MoveTestTactic::calculateNextIntent(IntentCoroutine::push_type &yield)
                                            Angle::zero(), 0.0, 0));
     } while ((this->robot->position() - this->destination).len() > 0.01);
 }
+
+void MoveTestTactic::accept(TacticVisitor &visitor) const
+{
+    // WARNING: Tactic Vistor does not implement visiting this Tactic
```

Comment:
Would be good to explain why we don't implement this here and in `accept` in `StopTestTactic` as well.

Reply:
done

Reply:
where's the explanation sorry?

Reply:
https://github.com/UBC-Thunderbots/Software/pull/854/files#diff-8addbc13c86a28250d4967f183b212b5
You're probably looking at an out of date file

Reply:
I'm confused, I was looking for an explanation of why we don't implement it, just to be clear? 

![Screenshot from 2019-10-07 20-21-25](https://user-images.githubusercontent.com/9075711/66364923-26399f00-e940-11e9-9897-13d30cfabd7b.png)

Reply:
Ah I completely missed the point of your comment. I'll add an explanation